Packages/Testing-MIES/UTF_BasicHardwareTests.ipf

```diff
@@ -1862,6 +1862,34 @@ Function StartDAQDuringTP_IGNORE(device)
 	wv[%$"Analysis function (generic)"][%Set] = "WriteIntoLBNOnPreDAQ"
 End
 
+Window AbortTP_dummy() : Panel
+	PauseUpdate; Silent 1		// building window...
+	NewPanel /W=(150,77,450,277) as "AbortTP_dummy"
+	SetWindow kwTopWin,userdata(random_change)=  "438.25"
+EndMacro
```

Comment:
Why do we need this macro?

Reply:
It was a residue from another test approach I missed to delete. I fixed the commit.